.travis.yml

```diff
@@ -1,184 +1,92 @@
-    - source activate test-environment
-    - conda install mkl blas=*=mkl numpy 'scipy<1.5' pytest pytest-cov cython coveralls
-    - python setup.py install --with-openmp
-  after_success:
-    - coveralls
+script:
+  - python -c "import qutip; qutip.about()"
+  - pytest --verbosity=1 --rootdir="${QUTIP_INSTALL}/tests" -c "${QUTIP_INSTALL}/tests/pytest.ini" --pyargs qutip
 
-_stage_osx_37: &stage_osx_37
+_mac_generic_setup: &mac_generic_setup
   os: osx
-  name: "Python 3.7, OSX 10.13, XCode 10"
-  osx_image: xcode10
   language: generic
-  # change dir
-  before_script:
-    - mkdir qutip_testing
-    - cd qutip_testing
-  # command to run tests
-  script:
-    - python -m qutip.about
-    - pytest --verbosity=1 --disable-pytest-warnings --cov=qutip --pyargs qutip
   install:
-    - wget https://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh -O miniconda.sh
-    - bash miniconda.sh -b -p $HOME/miniconda
-    - export PATH="$HOME/miniconda/bin:$PATH"
-    - hash -r
-    - conda config --set always_yes yes --set changeps1 no
-    - conda info -a
-    - conda create -q -n test-environment python=3.7
-    - source activate test-environment
-    # - conda install mkl blas=*=mkl numpy scipy pytest pytest-cov cython coveralls
-    - conda install nomkl blas=*=openblas numpy scipy pytest pytest-cov cython
+    - conda install nomkl blas=*=openblas numpy scipy matplotlib cython pytest
     - python setup.py install
 
-_stage_osx_38: &stage_osx_38
-  os: osx
-  name: "Python 3.8, OSX 10.13, XCode 10"
-  osx_image: xcode10
-  language: generic
-  # change dir
-  before_script:
-    - mkdir qutip_testing
-    - cd qutip_testing
-  # command to run tests
-  script:
-    - python -m qutip.about
-    - pytest --verbosity=1 --disable-pytest-warnings --cov=qutip --pyargs qutip -m "not slow"
-  install:
-    - wget https://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh -O miniconda.sh
-    - bash miniconda.sh -b -p $HOME/miniconda
-    - export PATH="$HOME/miniconda/bin:$PATH"
-    - hash -r
-    - conda config --set always_yes yes --set changeps1 no
-    - conda info -a
-    - conda create -q -n test-environment python=3.8
-    - source activate test-environment
-    # - conda install mkl blas=*=mkl numpy scipy pytest pytest-cov cython coveralls
-    - conda install nomkl blas=*=openblas numpy scipy pytest pytest-cov cython
-    - python setup.py install
-
-stages:
-  - test
-
 jobs:
   include:
-    - stage: test
-      <<: *stage_linux_38
-    - stage: test
-      <<: *stage_linux_omp
-    - stage: test
-      <<: *stage_linux_no_cython
-    - stage: test
-      <<: *stage_osx_37
-    - stage: test
-      <<: *stage_osx_38
+    - name: "Linux, Python 3.8: regular"
+      python: 3.8
+      install:
+        - conda install numpy scipy matplotlib pytest cython
+        - python setup.py install
+
+    - name: "Linux, Python 3.7: OpenMP, MKL, scipy<1.5 (coverage)"
```

Comment:
The name says scipy<1.5 but the installation does not set that limit.

Reply:
Thanks for spotting it!  I'll have made the exact same mistake in #1348 too (already merged to `dev.major`) so I'll send another quick PR on that branch to fix it there as well.